www/css/typography.css

```diff
@@ -0,0 +1,113 @@
+@import 'colors.css';
+
+body {
```

Comment:
I have not included this file in the app in this PR (it is included in the account redesign) but here it is to show kind of the gist of what I'm trying to pull out and make consistent